aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2175,10 +2163,6 @@ Tensor slice(
     auto quantizer = create_subtensor_quantizer(self, false, start_val, end_val, dim, step);
     result = as_strided_qtensorimpl(self, sizes, strides, storage_offset, quantizer);
   } else {
-    // NB: it is extremely important to perform a redispatch here for
```

Comment:
Why was this removed?

Reply:
whoops I think this comment was added after Joels pr I will add back